example/home/test/test_blog.py

```diff
@@ -426,8 +427,12 @@ def test_blog_embed(self):
             if block["blockType"] == "VideoBlock":
                 embed = block["youtubeLink"]
                 self.assertTrue(isinstance(embed["url"], str))
-                self.assertEquals(embed["embed"], raw_embed["html"])
-                self.assertEquals(embed["rawEmbed"], json.dumps(raw_embed))
+                if WAGTAIL_VERSION >= (2, 11):
+                    self.assertEquals(embed["embed"], raw_embed["html"])
+                    self.assertEquals(embed["rawEmbed"], json.dumps(raw_embed))
+                else:
+                    self.assertEquals(embed["embed"], None)
```

Comment:
question: why is this the case for <= 2.11?

could probably use a comment. future us will thank you ;)

Reply:
@zerolab I just remembered [this issue](https://github.com/wagtail/wagtail/issues/6656#issuecomment-749711713) and now I [added](https://github.com/GrappleGQL/wagtail-grapple/pull/176/commits/ff78bb127e6a30c42eb8effd6681bae0e04024d5) a custom `WAGTAILEMBEDS_FINDERS` setting when running with Wagtail version < 2.11 and now the tests pass as expected without the if statement.